packages/loader/container-loader/src/container.ts

```diff
@@ -1341,9 +1349,10 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         const storageService = await this.service.connectToStorage();
 
         this._storageService =
-            new RetriableDocumentStorageService(storageService, this.logger);
+            new RetriableDocumentStorageService(storageService, this.mc.logger);
 
         if(this.options.summarizeProtocolTree === true) {
+            this.mc.logger.sendTelemetryEvent({eventName:"summarizeProtocolTreeEnabled"});
```

Comment:
#nit This property doesn't seem to change within the lifetime of the container. Should we log this when we observe it for the first time? It doesn't really matter right now as this is called when we attach/load so there's no risk for sending too many events...

Reply:
in constructor i don't actually inspect the value, i just merge the config and the options, so i prefer to keep the check here, which will only be called once, and is a place where we actually care about the value.